<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #025ea1;</string>
  </property>
  <widget class="QPushButton" name="ans_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>391</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white; 
    padding: 20px 40px 20px 40px; 
    width: calc(100% - 80px); 
    cursor: pointer; 
    background-color: #025EA1; 
color: white;
text-align: left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ans_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="ans_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>391</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white; 
    padding: 20px 40px 20px 40px; 
    width: calc(100% - 80px); 
    cursor: pointer; 
    background-color: #025EA1; 
color: white;
text-align: left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ans_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="ans_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>391</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white; 
    padding: 20px 40px 20px 40px; 
    width: calc(100% - 80px); 
    cursor: pointer; 
    background-color: #025EA1; 
color: white;
text-align: left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ans_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="ans_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>391</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white; 
    padding: 20px 40px 20px 40px; 
    width: calc(100% - 80px); 
    cursor: pointer; 
    background-color: #025EA1; 
color: white;
text-align: left;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ans_group</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>171</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6CACE4; 
color: white;
transition-duration: 500ms; 
cursor: pointer; 
border-radius: 17px;
</string>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="question_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>ans_1</zorder>
  <zorder>ans_2</zorder>
  <zorder>ans_3</zorder>
  <zorder>ans_4</zorder>
  <zorder>next_btn</zorder>
  <zorder>question_text</zorder>
  <zorder>image</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ans_group"/>
 </buttongroups>
</ui>
